.github/workflows/deployment.yml

```diff
@@ -0,0 +1,59 @@
+# Copied from https://github.com/notlmn/browser-extension-template/blob/master/.github/workflows/deployment.yml
+name: Deployment
+
+on:
+  workflow_dispatch:
+  # You can manually trigger a deployment on GitHub.com
+  # https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
+  schedule:
+    - cron: '42 4 * * 4' # At 04:42 every Thursday
+
+jobs:
+  Build:
+    outputs:
+      shouldDeploy: ${{ steps.daily-version.outputs.created }} || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 20
+      - run: npm ci
```

Comment:
You dropped the `npm test // Includes the build` line; without this, there are no artifacts.

Reply:
Wait, I thought `npm ci` runs `npm test` as well. TIL!